File: app/Models/Ingrediente.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Ingrediente extends Model
{
    use HasFactory;

    protected $table = 'ingredientes';

    protected $fillable =
        [
            'id',
            'nombre',
            'descripcion',
            'created_at',
            'updated_at'
        ];
}

File: models/ingrediente.go
package models

import "time"

type Ingrediente struct {
	ID          uint      `json:"id" gorm:"primaryKey"`
	Nombre      string    `json:"nombre"`
	Descripcion string    `json:"descripcion"`
	CreatedAt   time.Time `json:"created_at"`
	UpdatedAt   time.Time `json:"updated_at"`
}

func (Ingrediente) TableName() string {
	return "ingredientes"
}
